inference-engine/src/transformations/src/transformations/op_conversions/convert_mvn1.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_mvn1.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/rt_info.hpp>
+
+#include <ngraph/opsets/opset2.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertMVN1, "ConvertMVN1", 0);
+
+ngraph::pass::ConvertMVN1::ConvertMVN1() {
+    auto mvn = pattern::wrap_type<opset2::MVN>(); // MVN was missing in opset1
+
+    ngraph::matcher_pass_callback callback = [](pattern::Matcher& m) {
+        auto mvn_node = std::dynamic_pointer_cast<opset2::MVN>(m.get_match_root());
```

Comment:
I think you should create variable match_root and use it instead of calling m.get_match_root().